js/dlc-exclusives.js

```diff
@@ -0,0 +1,132 @@
+DLC_EXCLUSIVES = {
```

Comment:
This is weird. It's a dictionary where every value is `true`, which is redundant.

This should be more idiomatically written as
```js
const DLC_EXCLUSIVES = new Set([
    "Dueling Shield",
    "Carian Thrusting Shield",
    // ...
]);
```

Reply:
Thanks. Using Set slipped my mind.